### ISSUE TITLE
Rendering backend exceptions and Twig errors in debug mode

### ISSUE THREAD

**Issue by weaverryan** [netgen-layouts/layouts-core#29]
Hi!

This is just an idea. I think it's actually 2 separate (but related things). Both relate to what happens when you make a coding mistake, and the result is that something "doesn't work" while in the layouts admin.

A) If any Ajax call fails with a 500 error, it'd be cool (in debug mode at least) to render the full exception in a modal. I did this recently - you could re-use the code: https://github.com/symfony/ux/pull/467

B) In some cases, if you have an error in Twig, the error is hidden entirely. I think this is to avoid error in production from a user error. But, for example, when creating the "preview" template for a content browser, any mistake I made in that Twig template wasn't shown to me. It'd be awesome to ALLOW the error to be thrown in debug mode and, along with (A), show it to the user.

Cheers!

**Comment by emodric** [comment]
Hi @weaverryan 

I remembered that we actually had this feature from the start, but due to a bug which I investigated today, it did not show up.

The error popup looks like this:

![image](https://user-images.githubusercontent.com/362286/197156880-56d163d6-1321-4e21-bd44-5fa819f83573.png)

This still does not solve the content browser preview issue. I will try to implement your suggestion with the popup there.

Thanks!
